src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-grid/dot-edit-layout-grid.component.html

```diff
@@ -1,6 +1,8 @@
 <div [ngGrid]="gridConfig" (onDragStop)="updateModel()" (onResizeStop)="updateModel()">
-    <dot-container-selector *ngFor="let gridBox of grid; let i = index" (change)="updateContainers($event)" (remove)="onRemoveContainer(i)"
-        [(ngGridItem)]="gridBox.config" [selectedContainersList]="gridBox.containers" [multiple]="true">
+    <div class="container-selector" *ngFor="let gridBox of grid; let i = index" [(ngGridItem)]="gridBox.config">
```

Comment:
This isn't a container selector this is a box, so class block should be like: "grid-box" or event just "box"